route/access_rules.go

```diff
@@ -47,14 +47,15 @@ func (t *Target) AccessDeniedHTTP(r *http.Request) bool {
 			// ensure we only get the ip string
 			xip = strings.TrimSpace(xip)
 			// only continue if xip differs from host
-			if xip != host {
-				if ip = net.ParseIP(xip); ip == nil {
-					log.Printf("[WARN] failed to parse xff address %s", xip)
-				}
-				// check xff source and return if denied
-				if t.denyByIP(ip) {
-					return true
-				}
+			if xip == host {
+				continue
+			}
+			if ip = net.ParseIP(xip); ip == nil {
+				log.Printf("[WARN] failed to parse xff address %s", xip)
```

Comment:
don't you need to continue here?

Reply:
The behavior of `denyByIP` is to immediately return false when passed a `nil` ... so it is effectively doing a continue.  However, I can see the point that adding a continue would be more explicit and be more future proof should additional code be added between this statement and the `denyByIP` call.